CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
 # Change Log (Release Notes)
 
 All _notable_ changes to this project will be documented in this file (`CHANGELOG.md`).
-This project adheres to [the Python form of Semantic Versioning](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme) (or at least we try).
+
+This project adheres to [Semantic Versioning 2](http://semver.org/spec/v2.0.0.html), the BigchainDB **public API**:
```

Comment:
My proposal is to change it back to saying that this project adheres to the Python form of Semantic Versioning.